notebooks/data-sources/TestGrid/metrics/metric_template.ipynb

```diff
@@ -369,6 +369,115 @@
     "    flat_list = [item for sublist in percent_label_by_grid_csv for item in sublist]\n",
```

Comment:
<p>These last few values appear to have the exact same duration. Can we confirm that this data is correct and  not duplicating values? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/297/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
Those values are from the same old function, so they haven't changed. We just added this function, no output results are here. We will use this function for our next issue- #282

Reply:
<p>But they have changed. The output is different. These values are all NaN in the previous version output, and now they are all 80.15. Can you confirm that this data is correct and not just duplicating values?</p>

Reply:
![image](https://user-images.githubusercontent.com/4494906/122988883-37268000-d370-11eb-80f9-945be41d0945.png)

Reply:
<p>so we reran the notebook for the same <code>testgrid_183.json</code>  data set that the notebook was running earlier and the <code>testgrid_labelwise_encoding</code>  function was also rerun exactly as before, but still seeing those last few values differ, not sure why this is happening...perhaps we can run it on the latest data set in ceph instead?</p>

Reply:
We have added a new updated version of `testgrid_labelwise_encoding` function. This contains a parameter `overall_only` which is by default True so that it doesn't affect the current processes. 

When we add the parameter, `overall_only = False`, we were able to see the changes in the value of `test_duration`, but not with `overall_only = True`.

All other modifications like removing some extra code for the label condition can be handled separately.

cc- @MichaelClifford @Shreyanand @chauhankaranraj